package.json

```diff
@@ -102,6 +102,7 @@
     "axios": "^0.20.0",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.0.6",
+    "basic-auth": "^2.0.1",
```

Comment:
There is no need for this dependency. Header is simply base64-encoded and can be decoded by `Buffer.from(data, 'base64').toString().split(':')`. First half is username while the later half is the password.

Reply:
as you prefer